STYLE.md

```diff
@@ -0,0 +1,90 @@
+# Travis CI documentation style guide
+
+## Markdown and structure
+
+We're planning to lint the Markdown with [Coala.io][coala] and the [MarkdownBear][bear].
+
+We'll be using a subset (TBD) of the full [list of checks][checks].
+
+[coala]:  http://coala.io/  "Coala CI"
+[bear]:   https://github.com/coala/bear-docs/blob/master/docs/MarkdownBear.rst  "MarkdownBear"
+[checks]: https://github.com/coala/bear-docs/blob/master/docs/MarkdownBear.rst#settings "MarkdownBear checks"
+
+### Headings
+
+For historical reasons, the top level heading in Jekyll markdown files is level 2 (##) not level 1 (#).
+
+We use ATX style headings, and do not use the optional closing hashes:
+
+```markdown
+## This is an H2
+
+### This is an H3
+
+#### This is an H4
+```
+
+We do not use underline style headings:
+
+```
+Do not use this style heading
+=============================
+```
+
+### TOC
+
+Add a table of contents to a page with the following html snipped:
+
+```html
+<div id="toc"></div>
+```
+
+### GUI
+
+Make sure all references to items in a GUI matches the case of the UI, and is marked with *asterisks*.
+
+### Links
+
+Make sure all links have titles:
+
+```markdown
+The [link][example1] in the text
+
+[example1]: http://www.example.com  "Example URL"
+```
+
+or
+
+```markdown
+The [link](http://www.example.com "Example URL") in the text
+
+```
+
+### Code Inline
+
+All function names, filenames, etc should be marked with `back-ticks`.
```

Comment:
Maybe this can be more precise.

With that the database PostgreSQL should not be backticked, but the `psql` application should.